test/entice/logic/movement_test.exs

```diff
@@ -1,10 +1,13 @@
 defmodule Entice.Logic.MovementTest do
   use ExUnit.Case, async: true
   alias Entice.Entity
+  alias Entice.Entity.Coordination
   alias Entice.Utils.Geom.Coord
-  alias Entice.Logic.Movement
+  alias Entice.Logic.{Movement, MapInstance}
   alias Entice.Logic.Player.Position
+  use Entice.Logic.Map
```

Comment:
`use`'s should be grouped together at the beginning, I'm encouraging to globally keep to this style btw: https://github.com/levionessa/elixir_style_guide